examples/widgets/meter/lv_example_meter_5.c

```diff
@@ -0,0 +1,378 @@
+#include "../../lv_examples.h"
+#if LV_USE_METER && LV_BUILD_EXAMPLES
+
+#define ARRAY_SIZE(x) (sizeof(x) / sizeof((x)[0]))
+
+typedef enum {
+    METER_SIMPLE,
+    METER_LINEAR,
+    METER_MULTI_ARCS,
+    METER_MULTI_ARCS_LINEAR,
+    METER_CLOCK,
+    METER_PIE_CHART,
+    METER_PIE_CHART_LINEAR,
+} meter_t;
+
+static struct meter_item {
+    lv_obj_t * obj;
+    lv_anim_t a;
+    lv_point_t coord;
+    uint16_t w;
+    uint16_t h;
+    uint8_t indic_cnt;
+    uint8_t reverse : 1;
+    uint8_t toggle_needle_side: 1;
+    uint8_t hide_borders : 1;
+    uint8_t meter_type: 3;
+    char * name;
+    lv_meter_indicator_t * indic[5];
+} meter_items[] = {
+    /*Horizontal meters*/
+    [0] = {
```

Comment:
Is it a really cross platform notion? I think
```
{...}, {...}, {...}
```
would be more portable.

Reply:
oh, my bad, GNU extension
fixed.